shopfloor_mobile/static/wms/src/scenario/zone_picking.js

```diff
@@ -540,6 +554,16 @@ const ZonePicking = {
                             this.odoo.call("scan_location", {barcode: scanned.text})
                         );
                     },
+                    on_unload_at_destination: () => {
+                        const loaded_data = this.state.data.buffer_is_loaded;
+                        this.odoo._update_headers(
```

Comment:
IMO you don't need this call to `_update_headers` because the headers will be inject automatically w/ values from `current_*` which you already overridden. 
If they don't get refreshed because `odoo` computed property is not refreshed I would create a new one instead:
```
            const odoo_params = this._get_odoo_params();
            const odoo = this.$root.getOdoo(odoo_params);
```

Reply:
The problem is `_get_odoo_params` relies on `current_picking_type` and `current_zone_location` and they would both need some re factoring only for this specific case. Knowing that they are used in other places in this module I found safer to not change them and do the `_update_headers` here.

Reply:
Then I prefer you go for
```
const odoo_params = this._get_odoo_params();
const odoo = this.$root.getOdoo(odoo_params);
```
w/out altering the other object

Reply:
Done that.